build.sbt

```diff
@@ -86,7 +86,7 @@ pomExtra := {
 libraryDependencies ++= Seq(
   "com.typesafe.akka" % "akka-testkit_2.10" % akkaVersion % "test",
   "io.spray" % "spray-testkit" % sprayVersion % "test",
-  "com.datastax.cassandra" % "cassandra-driver-core" % "2.0.9" % "test" exclude("com.google.guava", "guava"),
+  "com.datastax.cassandra" % "cassandra-driver-core" % "2.0.10-SNAPSHOT" % "test" exclude("com.google.guava", "guava"),
```

Comment:
Since HeartbeatVerificationTest depends on functionality from 2.0.10 (which is yet to be released), it might be a good idea to defer merging this until 2.0.10 is released (for convenience of running tests without needing to install the snapshot for everyone locally).